src/components/organisms/AssetActions/Consume.tsx

```diff
@@ -157,16 +157,23 @@ export default function Consume({
         <div className={styles.filewrapper}>
           <File file={file} />
         </div>
-        <div className={styles.pricewrapper}>
-          {isConsumable ? (
-            <Price ddo={ddo} conversion />
-          ) : (
-            <div className={styles.help}>
-              There is not enough liquidity in the pool to buy this data set.
-            </div>
-          )}
-          {!isInPurgatory && <PurchaseButton />}
-        </div>
+        {type === 'algorithm' ? (
```

Comment:
I think we can remove this condition again since we want to allow compute on an algorithm. So we do not specifically have to state that, it is implied by switching from `Download` to `Compute` component, just like with data sets.